benchmarks/benchmark_serving.py

```diff
@@ -711,9 +737,9 @@ def main(args: argparse.Namespace):
   parser.add_argument(
       "--request-rate",
       type=float,
-      default=float("inf"),
+      default=0.0,
```

Comment:
Why do we change it to 0.0?

Reply:
Similar to the boolean flag, if I am sweeping through different request-rate, I can not set float("inf"), because it is being recognized as string when passing in to benchmark_serving.py.